examples/supertuxkart/main.go

```diff
@@ -57,58 +69,118 @@ func main() {
 	cmd.Stdout = os.Stdout
 
 	if err := cmd.Start(); err != nil {
-		log.Fatalf(">>> Error starting cmd: %v", err)
+		log.Fatalf("error starting cmd: %v", err)
 	}
 
 	// SuperTuxKart refuses to output to foreground, so we're going to
 	// poll the server log.
-	ready := false
 	home, err := os.UserHomeDir()
 	if err != nil {
-		log.Fatalf(">>> Could not get home dir: %v", err)
+		log.Fatalf("could not get home dir: %v", err)
 	}
-	for i := 0; i <= 10; i++ {
-		time.Sleep(time.Second)
 
-		logFile, err := os.Open(path.Join(home, logLocation))
-		if err != nil {
-			log.Printf(">>> Could not open server log: %v", err)
-			continue
-		}
+	t := &tail.Tail{}
+	for i := 0; i < 10; i++ {
+		time.Sleep(time.Second)
 
-		content, err := ioutil.ReadAll(logFile)
+		t, err = tail.TailFile(path.Join(home, logLocation), tail.Config{Follow: true})
 		if err != nil {
-			log.Printf(">>> Could not read server file: %v", err)
+			log.Print(err)
 			continue
+		} else {
+			break
 		}
-		output := string(content)
-		println(">>> Output from server_config.log:")
-		print(output)
-
-		if strings.Contains(output, "Listening has been started") {
-			ready = true
-			log.Printf(">>> Moving to READY!")
+	}
+	defer t.Cleanup()
+	for line := range t.Lines {
+		fmt.Println(line.Text)
```

Comment:
why use `fmt` here instead of `log`?

Reply:
Since the line is coming directly from supertuxkart, it already has a prefix on it.  Adding the wrapper logging to the general logging makes for double prefixes and will muddy the logs.

The general intent is to keep the supertuxkart logs visible, while adding in additional logging from the wrapper

Reply:
That makes total sense. Can you add a comment above this line so that we don't lose that information after the PR merges?

Reply:
No problem.